.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+  # Enable version updates for GOLang dependencies
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "monthly"
```

Comment:
Is this file also needed for projects that already have dependabot installed?

Reply:
We definitely want this configured to update npm packages as well. This behavior has already been working on projects with dependabot enabled. Those projects are using dependabot to upgrade Go and npm deps without this file being present

Reply:
@mickmister Feel free to commit any changes to this PR if you prefer different functionality. Our goal is to provide the basic configuration, while leaving additional project-specific configuration to the developers

Reply:
> Is this file also needed for projects that already have dependabot installed?

@mickmister The[ ticket raised](https://github.com/mattermost/actions-workflows/pull/45), instructs to deploy dependabot to this repository. If this is not the case, let me know so that we can close this PR. 

Feel free to update the ticket, with an actual list, of projects needing depndabot installed,.